app/views/course/achievement/achievements/_achievement.html.slim

```diff
@@ -24,9 +24,13 @@
 
   td
     div.btn-group
-      - if can?(:manage, achievement)
+      - if can?(:award, achievement)
```

Comment:
Good catch >_< didn't notice that. In fact, hound recommends `conditions.empty?` instead. 

Fixed it!